src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+@Override
+public void doWork() {
+    System.out.println("Excavator do work");
+}
+@Override
+public void stopWork() {
+    System.out.println("Excavator stoped work");
```

Comment:
There is a typo in the string "Excavator stoped work". It should be "Excavator stopped work".